crates/api_common/src/tagline.rs

```diff
@@ -0,0 +1,52 @@
+use crate::sensitive::Sensitive;
+use lemmy_db_schema::source::tagline::Tagline;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ListTaglines {
+  pub auth: Option<Sensitive<String>>,
```

Comment:
This being the only API endpoint I see staying, and since it will only be used by admins, I'd make the auth mandatory.